src/Analysis.cpp

```diff
@@ -802,10 +813,381 @@ void CalledMethodsAnalysis::doAnalysis(Function &F, PointerAnalysis *PA, std::st
   }
 
   errs() << "\n\nRUNNING TESTS - ALLOWED_REDEFINE = " << ALLOW_REDEFINE << " TEST NAME - " << testName << "\n\n";
-  std::exit(runTests(ExpectedResult, PostCalledMethods));
+  // * Tests temporarily disabled; developing must call first and then changing structure 
+  // std::exit(runTests(ExpectedResult, PostCalledMethods));
+
+}
+
+// must call 
+void transfer(Instruction* I, SetVector<Instruction *>& workSet, std::map<std::string, MC>& mustCallSet, AliasMap& aliasedVars) {
```

Comment:
I think a lot of this function's code is shared with `transfer` in the called-methods analysis. Would it be possible to abstract out a "generic" transfer function that works for both of them, and have it call methods for the details that need to differ? That would be better software engineering than just copying code between the two.